tests/Microsoft.Fx.Portability.Tests/ApiPortServiceTests.cs

```diff
@@ -12,9 +12,9 @@
 
 namespace Microsoft.Fx.Portability.Tests
 {
-    public class ApiPortServiceTests
+    public sealed class ApiPortServiceTests : IDisposable
```

Comment:
I'm curious why this class is sealed but the others are not? :)

Reply:
This one creates a disposable member field - the sealed is so we don't need to implement the whole dispose pattern